app.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import 'highcharts';
+import ReactHighcharts from 'react-highcharts';
+
+class Chart extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: null,
+            error: null
+        };
+    }
+
+    componentDidMount() {
+        fetch("./psnrResults")
```

Comment:
I assume you're targeting chrome only for now (and maybe ever)? Not sure how well fetch plays now across all browsers.

Reply:
no edge, looks like, but most others are covered.  i think we can leave this and see if it becomes a problem.